### DISCUSSION TITLE
This template is a 1* place on Kaggle 🚀

### DISCUSSION THREAD

**Discussion by callmeBalloch** [ashleve/lightning-hydra-template#237] [Show and tell]
Hey guys just want to notice you that the template was used as part of the 1' place [Google Brain - Ventilator Pressure Prediction](https://www.kaggle.com/c/ventilator-pressure-prediction/leaderboard) solution. 
Here is the code: https://github.com/whoknowsB/google-brain-ventilator-pressure-prediction
Keep the good work!!!

**Comment by ashleve** [comment]
@whoknowsB Fantastic work, congrats on the win!
